bot.py

```diff
@@ -179,6 +181,16 @@ def _setup_api(self):
              print('Exception during print player profile')
         self.update_inventory();
 
+    def _setup_tree(self):
+        tasks = []
+
+        for taskConfig in self.config.tasks:
+            taskRef = globals()[taskConfig['type']]
```

Comment:
`globals()` is really bad behaviour. Please encapsulate properly. 

Also `taskConfig` is CamelCase and we use snake_case for variables and functions. CamelCase only for class definitions.